src/test/java/com/google/sps/UserTest.java

```diff
@@ -0,0 +1,89 @@
+package com.google.sps;
+
+import com.google.sps.user.User;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class UserTest {
+    private static final String ID = "userid";
+    private static final String EMAIL = "user@gmail.com";
+    private static final String NAME = "username";
+    private static final String SELF_INTRODUCTION = "I am the user";
+    private static final String IMG_URL = "/img.com";
+
+    @Test
+    public void getId() {
+        User newUser = new User.Builder(ID, EMAIL).build();
+        Assert.assertEquals(newUser.getId(), ID);
```

Comment:
Here is `assertEquals(result, expected)`, if there's an error, then the console will display something like:
expected : `result`. but was : `expected`
Change it to assertEquals(expected, result) to make it easier to read if a failure occurred. Modify the others also :) thanks.

Reply:
Changed. Thank you for the remark!